pinot-spi/src/main/java/org/apache/pinot/spi/utils/CommonConstants.java

```diff
@@ -359,6 +359,8 @@ public static class Broker {
 
     public static class Request {
       public static final String SQL = "sql";
+      public static final String V1SQL = "v1sql";
```

Comment:
Use official names ? `SSQE` and `MSQE`. Though I vote to drop the `Q` in the abbrevations.

Reply:
The official docs do use the v1 and v2 terminology as well - https://docs.pinot.apache.org/reference/single-stage-engine, https://docs.pinot.apache.org/developers/advanced/v2-multi-stage-query-engine and it seems more succinct and clear to say `v1sql` than something like `ssqeSql`.

Reply:
Ideally we should always try to use single-stage query engine and multi-stage query engine instead of v1 and v2, but we are so used to the latest that sometimes we _leak_ these terms.

Reply:
or use only `sse` and `mse` as the semantics of the fields are obvious and will be documented. `Sql` prefix is not required.

Reply:
My order of preference in descending order is:
1. `sqlV1` and `sqlV2` so they start with the same prefix of the default `sql`.
2. `v1Sql` and `v2Sql`
3. `sse` and `mse` sound a bit strange.